quinn-proto/src/connection/packet_builder.rs

```diff
@@ -138,12 +138,10 @@ impl PacketBuilder {
                 crypto.packet.local.tag_len(),
             )
         } else if space_id == SpaceId::Data {
-            let zero_rtt = conn.zero_rtt_crypto.as_ref().expect(
-                "sending packets in the application data space requires known 0-RTT or 1-RTT keys",
-            );
+            let zero_rtt = conn.zero_rtt_crypto.as_ref().unwrap();
             (zero_rtt.header.sample_size(), zero_rtt.packet.tag_len())
         } else {
-            unreachable!("tried to send {:?} packet without keys", space_id);
+            unreachable!();
```

Comment:
Are these changes related? They don't look like improvements to me.

Reply:
The verbose panic messages were moved into `poll_transmit`, where they precede this logic.

Reply:
Fair enough.